util/file.go

```diff
@@ -83,6 +84,7 @@ func GetPathRelativeTo(path string, basePath string) (string, error) {
 		return "", errors.WithStackTrace(err)
 	}
 
+	relPath = strings.Replace(relPath, `\`,"/", -1)
```

Comment:
Hm, I'm not sure this is the right place for the fix, nor quite the right fix. The bug, as [explained here](https://github.com/gruntwork-io/terragrunt/issues/89#issuecomment-269853544), is caused by the fact that Windows uses backslashes in its paths, but backslashes mean "escape" in HCL. So the fix is not to remove backslashes, but rather, to escape them, and to escape them *only* in the context of HCL usage.

Therefore, I think the proper fix is:

1. Create a method called like `cleanPathForHcl(path string)` that escapes backslashes (i.e. turns them into a double backslash). Add a comment with the explanation above.
1. Call that method at the end of the `findInParentFolders` and `pathRelativeToInclude` methods, since those methods are explicitly built to return valid HCL values.